Kentico/argallery-android
# DC-537 Introduce data model from JavaRX Delivery SDK

### Motivation

Which issue does this fix? Fixes #2

### Checklist


### How to test

Manual testing needed: 
* Try installing the app on the phone and test all of its functionality
* Add a new picture in the Kentico Cloud project, check whether the app retrieved the data correctly
